docs/client_advanced.rst

```diff
@@ -26,27 +26,35 @@ Custom Request Headers
 If you need to add HTTP headers to a request, pass them in a
 :class:`dict` to the *headers* parameter.
 
-For example, if you want to specify the content-type for the previous
-example::
+For example, if you want to specify the content-type directly::
 
-    import json
-    url = 'https://api.github.com/some/endpoint'
-    payload = {'some': 'data'}
-    headers = {'content-type': 'application/json'}
```

Comment:
Wrong way to specify content-type for this (JSON) example. It's obsolete.

Reply:
Agree.